### PR TITLE
Feature/add SW-VALUE-BLOCK-SIZE parsing

### PR DESCRIPTION
**Pull request by acomodi**
This PR adds parsing of the value block size tag. This refers to the dimension of a compound primitive with a VAL_BLK category

### CODE REVIEW

**Comment by acomodi** [comment]
[[cogu] investigate and add support for SW-VALUE-BLOCK-SIZE](https://trello.com/c/1PtOpXnl/782-cogu-investigate-and-add-support-for-sw-value-block-size)

**Review by andredami — APPROVED** [review]
LGTM

**Comment by acomodi** [comment]
[[autosar support] add support for VAL_BLK category and SW-VALUE-BLOCK-SIZE](https://trello.com/c/msy8ToZv/788-autosar-support-add-support-for-valblk-category-and-sw-value-block-size)

File: autosar/parser/parser_base.py [hunk 1]
```diff
@@ -272,6 +274,11 @@ def parseSwDataDefPropsConditional(self, xmlRoot):
                 swRecordLayoutRef = self.parseTextNode(xmlItem)
             elif xmlItem.tag == 'SW-CALPRM-AXIS-SET':
                 swCalprmAxisSet = self.parseSwCalprmAxisSet(xmlItem)
+            elif xmlItem.tag == 'SW-VALUE-BLOCK-SIZE':
+                swValueBlockSize = self.parseTextNode(xmlItem)
```
Comment (andredami) [inline autosar/parser/parser_base.py:278]: The `AUTOSAR_00051.xsd` schema states that the value is numerical 

```xml
<xsd:element maxOccurs="1" minOccurs="0" name="SW-VALUE-BLOCK-SIZE" type="AR:NUMERICAL-VALUE-VARIATION-POINT">
```
so I would force its casting to `int()`.

File: autosar/parser/parser_base.py [hunk 2]
```diff
@@ -272,6 +274,11 @@ def parseSwDataDefPropsConditional(self, xmlRoot):
                 swRecordLayoutRef = self.parseTextNode(xmlItem)
             elif xmlItem.tag == 'SW-CALPRM-AXIS-SET':
                 swCalprmAxisSet = self.parseSwCalprmAxisSet(xmlItem)
+            elif xmlItem.tag == 'SW-VALUE-BLOCK-SIZE':
+                swValueBlockSize = self.parseTextNode(xmlItem)
+            elif xmlItem.tag == 'SW-VALUE-BLOCK-SIZE-MULTS':
+                for sizeItem in xmlItem.findall('./*'):
+                    swValueBlockSizeMults.append(self.parseTextNode(sizeItem))
```
Comment (andredami) [inline autosar/parser/parser_base.py:281]: The `AUTOSAR_00051.xsd` schema states that each value is numerical 
```xml
<xsd:complexType>
   <xsd:choice maxOccurs="unbounded" minOccurs="0">
      <xsd:element name="NUMERICAL-VALUE-VARIATION-POINT" type="AR:NUMERICAL-VALUE-VARIATION-POINT"/>
   </xsd:choice>
</xsd:complexType>
```
so I would force its casting to `int()`.

**Review by andredami — CHANGES_REQUESTED** [review]
Minor comments.

Apart from those, successfully tested on XSD-compliant snippet:

```xml
<AUTOSAR xmlns="http://autosar.org/schema/r4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://autosar.org/schema/r4.0 0AUTOSAR_4-8-0.xsd">
    <AR-PACKAGES>
        <AR-PACKAGE>
            <SHORT-NAME>PackageDiProva</SHORT-NAME>
            <ELEMENTS>
                <APPLICATION-PRIMITIVE-DATA-TYPE>
                    <SHORT-NAME>AppPrimDt</SHORT-NAME>
                    <CATEGORY>VAL_BLK</CATEGORY>
                    <SW-DATA-DEF-PROPS>
                        <SW-DATA-DEF-PROPS-VARIANTS>
                            <SW-DATA-DEF-PROPS-CONDITIONAL>
                                <SW-VALUE-BLOCK-SIZE>12</SW-VALUE-BLOCK-SIZE>
                            </SW-DATA-DEF-PROPS-CONDITIONAL>
                        </SW-DATA-DEF-PROPS-VARIANTS>
                    </SW-DATA-DEF-PROPS>
                </APPLICATION-PRIMITIVE-DATA-TYPE>
                <APPLICATION-PRIMITIVE-DATA-TYPE>
                    <SHORT-NAME>AppPrimDtMulti</SHORT-NAME>
                    <CATEGORY>VAL_BLK</CATEGORY>
                    <SW-DATA-DEF-PROPS>
                        <SW-DATA-DEF-PROPS-VARIANTS>
                            <SW-DATA-DEF-PROPS-CONDITIONAL>
                                <SW-VALUE-BLOCK-SIZE-MULTS>
                                    <SW-VALUE-BLOCK-SIZE-MULT>1</SW-VALUE-BLOCK-SIZE-MULT>
                                    <SW-VALUE-BLOCK-SIZE-MULT>2</SW-VALUE-BLOCK-SIZE-MULT>
                                    <SW-VALUE-BLOCK-SIZE-MULT>3</SW-VALUE-BLOCK-SIZE-MULT>
                                </SW-VALUE-BLOCK-SIZE-MULTS>
                            </SW-DATA-DEF-PROPS-CONDITIONAL>
                        </SW-DATA-DEF-PROPS-VARIANTS>
                    </SW-DATA-DEF-PROPS>
                </APPLICATION-PRIMITIVE-DATA-TYPE>
            </ELEMENTS>
        </AR-PACKAGE>
    </AR-PACKAGES>
</AUTOSAR>
```

**Review by andredami — APPROVED** [review]
LGTM

**Pull request merged**